metrics.go

```diff
@@ -17,13 +17,18 @@ import (
 type MetricType string
 
 const (
-	builtinMetricsPrefix = "juju"
+	BuiltinMetricPrefix = "juju"
```

Comment:
Yes, it does, i think.. can used to compose the built-in metrics name in the core.